data/Localization/german_(germany)/global.ini

```diff
@@ -42851,7 +42858,7 @@ confirmkill_desc_0007=Ich habe gehört, dass ~mission(TargetName) tot sein könn
 confirmkill_from=~mission(Contractor)
 confirmkill_location_0001=die Überreste des Schiffes
 confirmkill_location_0002=die letzte Anlaufstelle
-confirmkill_location_0003=~mission(TargetName|NickOrFirst)'s letzte bekannte Position
+confirmkill_location_0003=~mission(TargetName|Last)'s letzte bekannte Position
```

Comment:
Moment
Shouldn't "NickOrFirst" be left as it is?

Reply:
Got changed in english version